corehq/apps/receiverwrapper/tests/test_submissions.py

```diff
@@ -166,7 +166,12 @@ def test_case_ledger_form(self):
         form_xml = self.get_xml('case_ledger_form')
         result = submit_form_locally(form_xml, domain=self.domain)
 
-        transaction = result.cases[0].get_transaction_by_form_id(result.xform.form_id)
+        open_cases = [case for case in result.cases if not case.closed]
+        closed_cases = [case for case in result.cases if case.closed]
+        self.assertEqual(len(open_cases), 1)
+        self.assertEqual(len(closed_cases), 1)
+
+        transaction = closed_cases[0].get_transaction_by_form_id(result.xform.form_id)
```

Comment:
This test was not making an assertion about open cases in the past. Is it necessary to add it now? This would be closer to what was being done before:

```py
# use tuple unpacking to verify single closed case
closed_case, = [case for case in result.cases if case.closed]
transaction = closed_case.get_transaction_by_form_id(result.xform.form_id)
```

Reply:
Updated: bf32a0b23a2